_docs/policies/submitting-policy.md

```diff
@@ -0,0 +1,80 @@
+# Purpose
+
+Policy Zero establishes the foundational guidelines for creating,
+reviewing, and maintaining policies in the `status-go` Git repository.
+This policy aims to create a collaborative, inclusive, and transparent
+process for defining repository policies, specifically regarding how
+developers engage with and contribute to the `status-go` repository.
+
+# Submitting a Policy Proposal
+
+- Any individual MAY propose a new policy.
+- Policy ideas SHOULD be discussed with contributors who wish to have
+a voice in how the repository operates, including Core Contributors
+(CCs) and external contributors.
+- All policies MUST be submitted to the `_docs/policies`
+ directory as a pull request (PR) within the `status-go` repository.
+- All policies MUST be in Markdown format.
+- Policy file names MUST follow [File name conventions for ADRs](https://github.com/joelparkerhenderson/architecture-decision-record?tab=readme-ov-file#file-name-conventions-for-adrs), e.g. `submitting-policy.md`.
+
+# Review and Approval Process
+
+The core function of the review and approval process for policy
```

Comment:
Ok that is fair. How about for this section I replace the description with a new item:

```
- Reviewers SHOULD keep in mind the principles communicated in 
[the README.md](README.md)
```

Reply:
Hm, perhaps we could include a mention of README.md under the Purpose section? For example: "For a more detailed description, please refer to [the README.md](README.md)." This would then implicitly apply to all points.